.goreleaser.yaml

```diff
@@ -14,7 +14,9 @@ before:
 builds:
   - id: darwin-build
     binary: infisical
-    ldflags: -X github.com/Infisical/infisical-merge/packages/util.CLI_VERSION={{ .Env.CLI_VERSION }}
+    ldflags: -X github.com/Infisical/infisical-merge/packages/util.CLI_VERSION={{ .Version }}
```

Comment:
Does .Version hold the tag by default? Any documentation on this

Reply:
btw that also allows you to test the goreleaser build locally, because you don't have to depend on any variable injection from github action :)

Reply:
nice

Reply:
I'm wondering how tag github tag maps to .Version. Where is the defined os is this parsed automatically and set to .Version

Reply:
It's part of goreleasers template magic :)
Actually I think it just reads the last git tag and removes the "v" prefix.